docs/knowledge-sharing/3 - Build AI App with Vercel SDK/material/content.md

```diff
@@ -0,0 +1,22 @@
+<!--- file: docs/knowledge-sharing/2 - Introduction to Ollama/material/content.md --->
+
+{% with pdf_file = "xxx.pdf" %}
```

Comment:
Do this need to be replace?

Reply:
I plan to link this with the deck (pdf) file.